src/mobster/cmd/upload/upload.py

```diff
@@ -218,6 +218,17 @@ def get_oidc_auth() -> OIDCClientCredentials | None:
             client_secret=os.environ["MOBSTER_TPA_SSO_TOKEN"],
         )
 
+    @staticmethod
+    def get_sbom_size(sbom_file: Path) -> int:
+        """
+        Args:
+            sbom_file: Absolute path to the SBOM file to upload
+
+        Returns:
+            int: size in bytes of the sbom_file provided
+        """
+        return sbom_file.stat().st_size
```

Comment:
Nitpick: Could we go with kilobytes at least? Bytes seems too small for most of them.

```suggestion
    @staticmethod
    def get_sbom_size_kilobytes(sbom_file: Path) -> int:
        """
        Args:
            sbom_file: Absolute path to the SBOM file to upload

        Returns:
            int: size in bytes of the sbom_file provided
        """
        return sbom_file.stat().st_size / 1024
```

Reply:
Sure, PR updated.